ttnn/cpp/ttnn/operations/conv/conv2d/device/conv2d_op_width_sharded_program_factory.cpp

```diff
@@ -303,6 +306,12 @@ operation::ProgramWithCallbacks multi_core_optimized_conv_width_sharded_v2_impl(
         "Number of Act Blocks along the Width {} should be divisible by the number of cores {}",
         num_blocks_act_w,
         input_num_cores);
+    bool packer_l1_acc_en = packer_l1_acc && ((has_bias && num_blocks_act_w > 1) || (num_blocks_act_w > 2));
```

Comment:
how is packer_l1_acc_en connected to bias?

Reply:
https://github.com/tenstorrent/tt-metal/blob/41c24698b9786c9dcf9b411ec630908abe19386b/ttnn/cpp/ttnn/operations/conv/conv2d/device/conv2d_op_sharded_program_factory.cpp#L1114

Reply:
It would be nice to not duplicate this noon obvious logic

Reply:
Done.